thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/grouping/AlertGrouperFactory.java

```diff
@@ -41,6 +41,8 @@ private static AlertGrouper fromStringType(String type) {
       return DUMMY_ALERT_GROUPER;
     case DIMENSIONAL:
```

Comment:
In brief, the difference between dimensional grouper and horizontal_dimensional grouper is that given the same string of dimensions "D1,D2", the former one groups anomaly in the leaf level (i.e., "D1,D2") and the latter one groups anomalies in the level of internal nodes (i.e., "D1" and "D2", respectively).

The details description for each grouper is provided in their JavaDoc.